infrasim/model.py

```diff
@@ -226,7 +226,7 @@ def init(self):
         self.__port = self.__chardev.get('port', self.__port)
         self.__wait = self.__chardev.get('wait', True)
         self.__path = self.__chardev.get('path')
-        self.__reconnect = self.__chardev.get('reconnect', 10)
+        self.__reconnect = self.__chardev.get('reconnect', 10) if self.__is_server is not True else None
```

Comment:
This could be better to read:

    if not self.__is_server:
        self.__reconnect = ...